src/context/UserProvider.tsx

```diff
@@ -174,6 +189,7 @@ export const UserProvider = (props: { children: ReactNode }) => {
 			value={{
 				state: {
 					user,
+					balance,
```

Comment:
this is user balance on login time and we didn't update it. maybe we need to update it periodically.